src/components/include/DefaultImportModalDialog.jsx

```diff
@@ -0,0 +1,323 @@
+        const { fileNames, formats, texts, dragging } = this.state;
+
+        const rows = fileNames.map((fileName, i) => ({
+            id: i,
+            fileName,
+            size: texts[i]?.size || "Not available",
+            lastModified: texts[i]?.lastModified || "Not available",
+            format: formats[i] || "Not available",
+        }));
+
+        const buttonContainerStyle = {
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "flex-end",
+            alignContent: "flex-end",
+        };
+
+        const columns = [
+            { field: "fileName", headerName: "File Name", flex: 1 },
+            { field: "format", headerName: "Format", flex: 1 },
+            { field: "lastModified", headerName: "Last Modified", flex: 1 },
+            {
+                field: "actions",
+                headerName: "Actions",
+                flex: 1,
+                sortable: false,
+                filterable: false,
+                disableColumnMenu: true,
+                renderHeader: () => (
+                    <div style={buttonContainerStyle}>
+                        <Button variant="contained" color="primary" size="small" component="label">
+                            Import
+                            <input
+                                style={{ display: "none" }}
+                                type="file"
+                                hidden
+                                multiple
+                                onChange={(event) => this.handleFileChange(event.target.files)}
+                            />
+                        </Button>
+                    </div>
+                ),
+                renderCell: (params) => (
+                    <div style={buttonContainerStyle}>
+                        <Button
+                            variant="contained"
+                            color="warning"
+                            size="small"
+                            onClick={() => this.handleFileRemove(params.row.fileName)}
+                        >
+                            Remove
+                        </Button>
+                    </div>
+                ),
+            },
+        ];
+
+        const dropZoneStyle = {
+            height: "160px",
+            width: "560px",
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            textAlign: "center",
+            justifyText: "center",
+            border: "2px dashed gray",
+            borderRadius: "5px",
+            cursor: "pointer",
+            backgroundColor: dragging && "grey",
+        };
+        const paperStyle = {
+            position: "absolute",
+            top: "10%",
+            width: "800px",
+        };
+
+        const dataGridStyle = {
+            height: "400px",
+            backgroundColor: dragging && "grey",
+        };
+
```

Comment:
This could be defined outside of the component

Reply:
I'll move it outside. btw, is it ok to have styles defined as js object, or is it better to move them to .scss files?

Reply:
It's OK to keep here for now